django/core/management/commands/makemigrations.py

```diff
@@ -248,12 +248,12 @@ def handle(self, *app_labels, **options):
                 else:
                     self.log("No changes detected")
         else:
+            if check_changes:
+                sys.exit(1)
```

Comment:
Should we also clarify a `help` for the `--check` argument? (as suggested in #15845 for the `migrate` command). For example:
```python
help="Exit with a non-zero status if model changes are missing migrations and don't actually write new migrations.",
```

Reply:
Good idea. I echoed the wording from the `--dry-run` option.